.github/workflows/release.yml

```diff
@@ -248,15 +302,16 @@ jobs:
       - name: Install dependencies and pyogrio wheel
         shell: bash
         run: |
-          python -m pip install -r ci/requirements-wheel-test.txt
+          uv pip install --no-cache -r ci/requirements-wheel-test.txt
```

Comment:
Is the `--no-cache` needed here?

Reply:
I think I was running into issues previously with using cached dependencies, but it might only be necessary when installing pyogrio itself.  Will test.